mypyc/irbuild/specialize.py

```diff
@@ -57,6 +57,25 @@
 specializers: Dict[Tuple[str, Optional[RType]], List[Specializer]] = {}
 
 
+def get_specialization(builder: 'IRBuilder', expr: CallExpr, callee: RefExpr,
```

Comment:
The name was a little confusing at first. I think `perform_specialization` or `apply_specialization` would be clearer. The current name sounds to be like it might return the specializer instead of calling it.

Reply:
done